index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Figma Template 1</title>
+    <link rel="stylesheet"
+          href="https://fonts.googleapis.com/css2?family=Crimson+Pro">
+    <link rel="stylesheet" href="style.css">   
+</head>
+<body>
+    <header>
+        <div class="logo"><a>Leon Espitia</a></div>
```

Comment:
- [ ] Please remove the Wrapping `<div>` tag and put your logo link `<a>Leon Espitia</a>` directly on your header for semantic reasons.